packages/story-editor/src/components/library/panes/media/local/dropDownMenu.js

```diff
@@ -134,7 +144,9 @@ function DropDownMenu({
               aria-expanded={isMenuOpen}
               aria-owns={isMenuOpen ? listId : null}
               id={buttonId}
-            />
```

Comment:
So this one, the dots are just smaller! 

This one is probably good to have some designer eyes on (@aaskedall, this look right? Matches the `Dots` in the Design System that was exported from Figma but still: https://google.github.io/web-stories-wp/storybook/?path=/story/designsystem-icons--all-icons)). 

Assuming we want to proceed. Let's do the following here. Remove the `attrs` from the `More` button, we've already got a bunch of props getting passed here so let's keep them all in the same place and put the button props here instead. 
```
<MoreButton 
...
     variant={BUTTON_VARIANTS.CIRCLE}
     size={BUTTON_SIZES.SMALL}
      type={BUTTON_TYPES.SECONDARY}
    >
``` 



| My demo | Staging | 
| --- | --- | 
| <img width="129" alt="Screen Shot 2021-08-16 at 3 44 24 PM" src="https://user-images.githubusercontent.com/10720454/129638300-69c444d8-59fd-48ff-bc7b-d8d7e917ebd7.png"> |  <img width="163" alt="Screen Shot 2021-08-16 at 3 45 33 PM" src="https://user-images.githubusercontent.com/10720454/129638327-73a80a58-220e-4968-b141-08d4993d621b.png"> |